persistence/sql/persister_consent.go

```diff
@@ -438,3 +438,92 @@ func (p *Persister) VerifyAndInvalidateLogoutRequest(ctx context.Context, verifi
 		return nil
 	})
 }
+
+func (p *Persister) FlushInactiveLoginConsentRequests(ctx context.Context, notAfter time.Time) error {
```

Comment:
So all the dockertests happen in one place via `persister_test.go`? I have implemented the dockertests + sqlite inside the `handler_janitor_test.go` file (https://github.com/Benehiko/hydra/blob/25a12c384f06785a3765cb361585a256aef28d46/cmd/cli/handler_janitor_test.go#L233)

Reply:
So it would be something similar to this test here - https://github.com/ory/hydra/blob/a8ca333bb65fe591d1795ce5b33760c97ff54d65/oauth2/fosite_store_helpers.go#L177-L176

So should I then move all the tests to helper methods and call them in `manager_test_helper.go`
https://github.com/ory/hydra/blob/e17a074774eafc93f4cb5620f2f186e87a9ec96e/consent/manager_test_helpers.go#L261

Reply:
Yup